cadc-storage-adapter/src/main/java/org/opencadc/inventory/storage/StorageAdapter.java

```diff
@@ -122,6 +122,9 @@ public void get(StorageLocation storageLocation, InputStreamWrapper wrapper, Set
      * Write an artifact to storage.
      * The value of storageBucket in the returned StorageMetadata and StorageLocation can be used to
      * retrieve batches of artifacts in some of the iterator signatures defined in this interface.
+     * The bucket value in the supplied artifact may be used by the storage implementation
+     * to organize files.  Batches of artifacts can be listed by bucket in two of the
+     * iterator signatures in this interface.
```

Comment:
Should clarify that if storageBucket is null in the returned StorageLocation, then the caller will never be able to do batched (bucket-based) validation via the iterator(String) method.

The comment that "Artifact.uriBucket may be used" should come with the caveat that rename will change that bucket value (assuming we stick with uriBucket instead of pkBucket - TBD)... which implies that the StorageAdapter.rename(...) method probably needs to be specified clearly to go with any mention of the artifact bucket scheme.

Reply:
Okay.  I'm going to leave it as is until we decide what ID uriBucket is based on.  The implementation is currently assuming pkBucket.